src/WebServer.ino

```diff
@@ -204,9 +204,15 @@ void sendContentBlocking(String& data) {
 void sendHeaderBlocking(bool json) {
   checkRAM(F("sendHeaderBlocking"));
   WebServer.client().flush();
+  String contenttype;
+  if (json)
+    contenttype = F("application/json");
+  else
+    contenttype = F("text/html");
+
 #if defined(ESP8266) && defined(ARDUINO_ESP8266_RELEASE_2_3_0)
   WebServer.setContentLength(CONTENT_LENGTH_UNKNOWN);
-  WebServer.sendHeader(F("Content-Type"), json ? F("application/json") : F("text/html"), true);
+  WebServer.sendHeader(F("Content-Type"), contenttype, true);
   WebServer.sendHeader(F("Accept-Ranges"), F("none"));
   WebServer.sendHeader(F("Cache-Control"), F("no-cache"));
   WebServer.sendHeader(F("Transfer-Encoding"), F("chunked"));
```

Comment:
@TD-er chunked here and not for other cores?

Reply:
Core 2.3.0 does not have support for chunked transfer, so we had to supply the headers ourselves.

The rest is sending the proper headers when using chunked transfer.